dbt/models/default/schema.yml

```diff
@@ -151,6 +303,270 @@ models:
+      - name: nearest_park_dist_ft
+        description: Distance in feet to the nearest park
+        meta:
+         notes: Pull via OpenStreetMap, key=leisure, value=park
+      - name: nearest_metra_stop_dist_ft
+        description: Distance in feet to the nearest Metra stop
+        meta:
+         notes: Taken from Metra GTFS feeds retrieved via transitfeeds.com
+      - name: nearest_metra_route_dist_ft
+        description: Distance in feet to the nearest Metra tracks
+        meta:
+         notes: Taken from Metra GTFS feeds retrieved via transitfeeds.com
+      - name: nearest_major_road_dist_ft
+        description: Distance in feet to the nearest major road/highway
+        meta:
+         notes: Pulled via OpenStreetMap, key=highway, value=motorway,primary,trunk
+      - name: nearest_hospital_dist_ft
+        description: Distance in feet to the nearest hospital (polygon)
+        meta:
+         notes: Taken from Cook County GIS shapefiles
+      - name: nearest_golf_course_dist_ft
+        description: Distance in feet to the nearest golf course (polygon)
+        meta:
+         notes: Taken from Cook County GIS shapefiles and OpenStreetMap
+      - name: nearest_cta_stop_dist_ft
+        description: Distance in feet to the nearest CTA stop
+        meta:
+         notes: Taken from CTA GTFS feeds retrieved via transitfeeds.com
+      - name: nearest_cta_stop_data_year
+        description: Distance in feet to the nearest Metra stop
+        meta:
+         notes: Taken from Metra GTFS feeds retrieved via transitfeeds.com
+      - name: nearest_cta_route_dist_ft
+        description: Distance in feet to the nearest CTA tracks
+        meta:
+         notes: Taken from CTA GTFS feeds retrieved via transitfeeds.com
+      - name: nearest_cemetery_dist_ft
+        description: Distance in feet to the nearest cemetery (polygon)
+        meta:
+         notes: Taken from Cook County GIS shapefiles
+      - name: nearest_bike_trail_dist_ft
+        description: Distance in feet to the nearest bike trail (linestring)
+        meta:
+         notes: Taken from Cook County GIS shapefiles
+      - name: nbhd_code
+        description: Assessor neighborhood
+        meta:
+         notes: First 2 digits are township code, last 3 digits are neighborhood code
+      - name: misc_unincorporated_area_bool
+        description: Indicator for a PIN within an unincorporated part of Cook County
+        meta:
+         notes: Taken from Cook County GIS shapefiles
+      - name: misc_subdivision_id
+        description: Subdivision ID for a given PIN
+        meta:
+          notes: Nearly all PINs are within one subdivision. Each subdivision has a unique ID with the PageSubref column. Page being the area and section and subref being the letter of that sub on a page. The subrefs go A B C D and so on, however new subs can kill old subs, so a page could have subrefs B C G M L R S T, with each letter being a unique subdivision
```

Comment:
Good example of a super long and ugly line in the docs. I couldn't get it to parse with line breaks no matter what I tried. 

**question:** If we aren't going to parse these programmatically (i.e. to build a separate data dictionary), maybe it's worth just moving everything into the `description` field, separating the summary like this:

```suggestion
      - name: misc_subdivision_id
        description: | 
        Subdivision ID for a given PIN
        
        Nearly all PINs are within one subdivision. Each subdivision has a unique ID with the PageSubref column. Page being the area and section and subref being the letter of that sub on a page. The subrefs go A B C D and so on, however new subs can kill old subs, so a page could have subrefs B C G M L R S T, with each letter being a unique subdivision
```

Or just with a period:

```suggestion
      - name: misc_subdivision_id
        description: | 
        Subdivision ID for a given PIN. Nearly all PINs are within one subdivision. Each subdivision has a unique ID with the PageSubref column. Page being the area and section and subref being the letter of that sub on a page. The subrefs go A B C D and so on, however new subs can kill old subs, so a page could have subrefs B C G M L R S T, with each letter being a unique subdivision
```

@jeancochrane What do you think?

Reply:
This is definitely a bummer @dfsnow! I played around with it a little bit too, and it seems like whatever Angular code they're using to generate the docs site is creating the `Columns` table summary by just concatenating everything from the `description` field into a single string. It's nice that the multiline formatting makes the detail view look better once you click on a row, but I wish the summary view would make use of that formatting too.

After trying a few things, I think my favorite is actually a combination of the two approaches you suggested -- period plus newline:

```suggestion
      - name: misc_subdivision_id
        description: |
        Subdivision ID for a given PIN.
        
        Nearly all PINs are within one subdivision. Each subdivision has a unique ID with the PageSubref column. Page being the area and section and subref being the letter of that sub on a page. The subrefs go A B C D and so on, however new subs can kill old subs, so a page could have subrefs B C G M L R S T, with each letter being a unique subdivision.
```

To me, this formatting gives the best experience for both the summary and detail views, since the period makes the summary easier to read while the newline makes the details easier to read:

![Screenshot 2023-09-12 112509](https://github.com/ccao-data/data-architecture/assets/14170650/2aa90c05-93d6-4ab4-82be-788e247e2919)
![Screenshot 2023-09-12 112606](https://github.com/ccao-data/data-architecture/assets/14170650/93200bdc-ac55-4b79-8afe-11f553632794)

(I don't love the blank `Details` line plus horizontal scrollbar in the details view, but based on the raw HTML I think that's an artifact generated by the fact that other rows have a `meta.notes` attribute while `misc_subdivision_id` does not, and I expect it to go away if and when we strip out `meta.notes` from the other rows.)

Does this sound reasonable to you? If so, I'm curious whether you think we should make this change now, or put it on the backburner until you flesh out the rest of the docs? It doesn't seem like it would make sense to make this change for fields that currently have a `meta.notes` attribute but no `description`, but I could be convinced otherwise.

Reply:
@jeancochrane This totally makes sense to me. Adding the period also makes the descriptions machine-parseable in case we ever need to split them between summary and details. IMO, let's just move everything into the description field, with line breaks for fields that have `meta.notes` only. I'll populate the summary line when I flesh out the docs.

Reply:
Sounds good @dfsnow, done in [0c6c213](https://github.com/ccao-data/data-architecture/pull/118/commits/0c6c213fa90cb9bc5b42162ce3fd3f3bc9cf4fbf)!